site/themes/embark/languages/en.yml

```diff
@@ -1,16 +1,277 @@
+      referal: Embark is a fully open source effort. Get involved and be part of the journey.
+      link: Join the Community
+
+  newsletter_section:
+    heading:
+      title: Sign up for updates
+      text: Sign up to our newsletter and get updates about Embark and our development progress first!
+    button:
+      label: Subscribe
+
+community_page:
+  header:
+    title: Join the growing community
+    tagline: Get support, give support and help building Embark.
+    link: Join the conversation
+
+  spotbox:
+    title: Checkout the GitHub Repos
+    text: Look for issues with specifics tags you may be interested in.
+    link: View repositories
+
+  contribute_section:
+    heading:
+      title: <span class="u-text-light">How to</span> contribute
+      referal: There are a number of ways to contribute and getting involved, from submitting a pull request to growing the community.
+
+  not_just_code_section:
+    heading:
+      title: It's not just <span class='u-text-light'>code</span>
+      referal: We believe that contributions go far beyond working on our code base. Help out with writing docs, creating a tutorial or simply talk to us on our channels.
+      link: Join us in Gitter
+
+    box_1:
+      title: Help improving the docs
+      text: You don't have to code to contribute, we're happy about any improvements to our documentation.
+
+    box_2:
+      title: Create a tutorial
+      text: Record a video of a simple tutorial and share it with the community.
+
+    box_3:
+      title: Help answer questions
+      text: Help out newcomers with questions in our Gitter channel or Stack Overflow.
+
+    box_4:
+      title: Create a plugin
+      text: Embark integrates well with other tools. Build a plugin to get support for your tool of choice.
+
+    box_5:
+      title: Report a bug
+      text: If you find a bug, let us know and we work on fixing it.
+
+    box_6:
+      title: Work on new features
+      text: Collaborate with peers on new features you want to see landing.
+
+  team_section:
+    heading:
+      title: Meet the <span class="u-text-ghost">team</span>.
+
+    job_team_lead: Creator and Team Lead
+    job_engineer: Software Engineer
+
+plugins_page:
+  header:
+    title: Plugins
+    tagline: Extend Embark to your needs.
+
+  intro_section:
+    heading:
+      referal: Plugins extend Embark and provide integration with other tools and technologies.
+      link: Build your own!
+
+  learn_more_link: Learn More
+
+  plugins:
+    bamboo:
+      desc: Plugin to add Bamboo support to embark
+    solc:
+      desc: Uses the command line Solidity Compiler for light-speed compile times
+    solium:
+      desc: The official Solium Plugin for the Embark Framework
+    verifier:
+      desc: Plugin for Embark to flatten and verify contracts on Etherscan
+    status:
+      desc: Status plugin to interact with embark
+    remix:
+      desc: Plugin to start Remix and Remix directly with Embark
```

Comment:
`Remix and Remix`? Not sure if we're missing a word or have just duplicated Remix

Reply:
No idea either haha, I've just taken what's already there and didn't even notice!